tests/integration/kubernetes/k8s-sandbox-vcpus-allocation.bats

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+load "${BATS_TEST_DIRNAME}/../../common.bash"
+load "${BATS_TEST_DIRNAME}/tests_common.sh"
+
+setup() {
+	[ "${KATA_HYPERVISOR}" == "dragonball" ] && \
```

Comment:
we need something more than dragonball to identify runtime-rs as it now supports clh and qemu/fc are on the way.

Reply:
Perfect, and this will end up covering more than just this test.
Thanks for the review, @bergwolf!